includes/class-wc-query.php

```diff
@@ -758,21 +755,22 @@ public static function get_main_search_query_sql() {
 	public static function get_layered_nav_chosen_attributes() {
 		if ( ! is_array( self::$_chosen_attributes ) ) {
 			self::$_chosen_attributes = array();
-			$attribute_taxonomies     = wc_get_attribute_taxonomies();
 
-			if ( ! empty( $attribute_taxonomies ) ) {
-				foreach ( $attribute_taxonomies as $tax ) {
-					$attribute    = wc_sanitize_taxonomy_name( $tax->attribute_name );
-					$taxonomy     = wc_attribute_taxonomy_name( $attribute );
-					$filter_terms = ! empty( $_GET[ 'filter_' . $attribute ] ) ? explode( ',', wc_clean( wp_unslash( $_GET[ 'filter_' . $attribute ] ) ) ) : array(); // WPCS: sanitization ok, input var ok, CSRF ok.
+			if ( ! empty( $_GET ) ) { // WPCS: input var ok, CSRF ok.
+				foreach ( $_GET as $key => $value ) { // WPCS: input var ok, CSRF ok.
+					if ( 0 === strpos( $key, 'filter_' ) ) {
```

Comment:
I'm sorry but I don't follow. Could you please elaborate?

Reply:
Instead of wrapping code in if statements, it's preferable to do the comparison early. e.g. 

```
if ( 0 !== strpos( $key, 'filter_' ) ) {
  continue;
}
```

it's a minor thing. Doesn't block this.

Reply:
In the interest of future changes that I make, preferable in which sense? Performance? Readability? Thanks.

Reply:
Readability due to less indentation :)